### ISSUE TITLE
Dropdown not showing items when being used in a modal dialog

### ISSUE THREAD

**Issue by sfrom** [IBM/carbon-components-angular#1816]
Tested with carbon componts 10.34.0 and carbon components angular 4.48.3

When opening a dropdown in a modal dialog the item list is not shown. It makes no difference if the list opening direction is up or down. I have to add the following css to make it work.

.bx--modal-content{
  overflow: visible;
}

.bx--modal-container{
  overflow: visible;
}

 I have tested react and vue dropdown component and they work as expected.

**Comment by cal-smith** [comment]
This may be another upstream regression 😞 If you inspect the DOM is the menu shown somewhere near the body?

**Comment by sfrom** [comment]
Yes, the menu is shown in the DOM just above the closing </body> tag.

![image](https://user-images.githubusercontent.com/8976083/118244911-652ac180-b4a0-11eb-9658-03057558a070.png)

**Comment by cal-smith** [comment]
If you edit that style for the menu and add an extremely high `z-index` does that fix the issue?
